### PR TITLE
Cm 7 Post Component

### PR DESCRIPTION
**Pull request by ramazanbulut**
Proposed changes
----------------

Post Component Added and some navigation configurations added to navigation to corresponding pages

Types of changes
----------------

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

Checklist
---------

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [ ] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

Further comments
----------------

(Write your answer here.)

### CODE REVIEW

File: Columbus/mobile/Columbus/package-lock.json [hunk 1]
```diff
@@ -2322,6 +2322,14 @@
         "@react-aria/utils": "^3.3.0"
       }
     },
+    "@react-native-async-storage/async-storage": {
```
Comment (onurcanavci) [inline Columbus/mobile/Columbus/package-lock.json:2325]: kanka package-lock.json dosyasını githuba yüklemeyelim, ben bu https://github.com/bounswe/2021SpringGroup7/pull/351 pr da package-lock ve yarn.lock dosyalarını sildim. Localinde package-lock dosyasını silip commit atman yeterli olacaktır.

File: Columbus/mobile/Columbus/src/components/PostCard/PostCard.js [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import {TouchableOpacity, View} from 'react-native';
+import ImageCarousel from './ImageCarousel';
+import UserInfo from './UserInfo';
+import LocationInfo from './LocationInfo';
+import PostTimeInfo from './PostTimeInfo';
+import Tags from './Tags';
+import PostingTime from './PostingTime';
+
+const PostCard = props => {
+  const postData = {
```
Comment (onurcanavci) [inline Columbus/mobile/Columbus/src/components/PostCard/PostCard.js:24]: Bunları da PostCard folderının altında PostCard.constants.js diye bir file a koysakdaha iyi olur

File: Columbus/mobile/Columbus/src/components/PostCard/PostTimeInfo.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import React, {Component} from 'react';
+import {Text, View} from 'native-base';
+
+export class PostTimeInfo extends Component {
```
Comment (onurcanavci) [inline Columbus/mobile/Columbus/src/components/PostCard/PostTimeInfo.js:4]: Bunu function yapsak nasıl olur? 
```
const PostTimeInfo = (props) => {
    const [data, setData] = useState(props.data);

  return(
   <Text......
 )
}
```
gibi

File: Columbus/mobile/Columbus/src/components/PostCard/PostingTime.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React, {Component} from 'react';
+import {Text} from 'native-base';
+
+export class PostingTime extends Component {
```
Comment (onurcanavci) [inline Columbus/mobile/Columbus/src/components/PostCard/PostingTime.js:4]: Burayı da aynı şekilde

File: Columbus/mobile/Columbus/src/components/PostCard/Tags.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  Text,
+  Center,
+  HStack,
+  Stack,
+  NativeBaseProvider,
+  Avatar,
+  Tag,
+} from 'native-base';
+
+export class Tags extends Component {
```
Comment (onurcanavci) [inline Columbus/mobile/Columbus/src/components/PostCard/Tags.js:16]: Burada aynı şekilde, class component artık çok tercih edilmiyor. Functional yazsak daha güzel olur

File: Columbus/mobile/Columbus/src/navigation/Navigation.js [hunk 6]
```diff
@@ -54,6 +65,6 @@ const AuthNavigation = () => (
 );
 
 export default function Navigation() {
-  const {user} = useAuth();
-  return user?.isAuthenticated ? <BottomTabNavigation /> : <AuthNavigation />;
+  const user = true;
```
Comment (onurcanavci) [inline Columbus/mobile/Columbus/src/navigation/Navigation.js:68]: kanka burayı da commitlerken dahil etmesen güzel olur aslında, sadece lokalinde böyle çalışıp bunu commitlemesen nasıl olur?

Comment (ramazanbulut) [reply Columbus/mobile/Columbus/src/navigation/Navigation.js:68]: tamamdir

File: Columbus/mobile/Columbus/src/components/PostCard/PostCard.js (hunk 2 above)
Comment (ramazanbulut) [reply Columbus/mobile/Columbus/src/components/PostCard/PostCard.js:24]: bi commit daha gerekcek gibi duruyor zaten hepsini halledip commitlicem pr uzerine

**Review by onurcanavci — APPROVED** [review]
Conflicts resolved

**Pull request merged**